### PR TITLE
Adding a new feature request template to our repo

### PR DESCRIPTION
**Pull request by gpanshu**
*Issue #, if available:* Chore

*Description of changes:*

Adding a new feature request template to our repo

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
Screen Shot 2022-04-05 at 3 52 06 PM
Screen Shot 2022-04-05 at 3 52 16 PM
Screen Shot 2022-04-05 at 3 52 25 PM
Screen Shot 2022-04-05 at 3 52 32 PM

### CODE REVIEW

**Review by div5yesh — COMMENTED** [review]
Can you add a screenshot on how the form looks?

**Comment by gpanshu** [comment]
> Can you add a screenshot on how the form looks?

Done

File: .github/ISSUE_TEMPLATE/2.feature_report.yaml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      required: true
+  - type: textarea
+    attributes:
+      label: Initialization steps (if applicable)
+      description: |
+        How do you initialize/setup this feature? Please walk us through it step by step. Screenshots can be attached in textarea below.
+      placeholder: |
+        1. Install '...'
+        2. Configure '...'
+        3. initialize at '...'
```
Comment (eeatonaws) [inline .github/ISSUE_TEMPLATE/2.feature_report.yaml:133]:
```suggestion
        3. Initialize '...'
```

File: .github/ISSUE_TEMPLATE/2.feature_report.yaml [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+        ```
+  - type: markdown
+    attributes:
+      value: |
+        ## Configuration
+  - type: textarea
+    attributes:
+      label: amplifyconfiguration.json
+      description: If applicable to the new feature, please provide your `amplifyconfiguration.json` file.
```
Comment (eeatonaws) [inline .github/ISSUE_TEMPLATE/2.feature_report.yaml:154]:
```suggestion
      description: If applicable to the new feature, please provide your `amplifyconfiguration.json` file with sensitive information removed.
```

**Review by eeatonaws — APPROVED** [review]

**Comment by div5yesh** [comment]
Why do I see duplicate fields for `Language and Async Model`? Maybe just issues with the screenshot?

**Comment by gpanshu** [comment]
> Why do I see duplicate fields for `Language and Async Model`? Maybe just issues with the screenshot?

Thats right. Its a problem with the screenshot. You can see the entire form here : https://github.com/aws-amplify/amplify-android/blob/chore/feature-request-template-update/.github/ISSUE_TEMPLATE/2.feature_report.yaml

**Review by div5yesh — APPROVED** [review]

**Pull request merged**